flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/operators/aggregate/BytesHashMapTest.java

```diff
@@ -98,7 +98,6 @@ public void testHashSetMode() throws IOException {
 		MemoryManager memoryManager = MemoryManagerBuilder
 			.newBuilder()
 			.setMemorySize(numMemSegments * PAGE_SIZE)
-			.setNumberOfSlots(32)
```

Comment:
I would merge this commit with the one removing the numberOfSlots argument. Makes it much clearer that this commit does affect behavior.